File: Android/app/src/main/java/knk/ee/neverland/pojos/Quest.java
package knk.ee.neverland.pojos;

import java.time.LocalDateTime;

public class Quest {
    private int id;
    private String name;
    private String description;
    private User creator;
    private LocalDateTime timeCreated;
    private int groupID;

    public String getName() {
        return name;
    }

    public String getDescription() {
        return description;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getGroupID() {
        return groupID;
    }

    public void setGroupID(int groupID) {
        this.groupID = groupID;
    }
}

File: Android/app/src/main/java/knk/ee/neverland/models/Quest.kt
package knk.ee.neverland.models

import com.google.gson.annotations.SerializedName
import org.joda.time.LocalDateTime

data class Quest(
    val id: Int,

    @SerializedName("title")
    val title: String,

    @SerializedName("desc")
    val description: String,

    @SerializedName("author")
    val author: User,

    @SerializedName("time_created")
    val timeCreated: LocalDateTime,

    @SerializedName("time_taken")
    var timeTaken: LocalDateTime?,

    val groupID: Int
)
